examples/plot_connectivity.py

```diff
@@ -52,9 +52,10 @@
 ###############################################################################
 # We can modify the connectivity list to test the effect of different
 # connectivity patterns. For example, we can remove all layer 2 inhibitory
-# connections.
+# connections. In the default network, the src_gids of each connection are
+# all of the same type so we'll just check the first element.
 new_connectivity = [conn for conn in net.connectivity
-                    if conn['src_type'] != 'L2_basket']
+                    if net.gid_to_type(conn['gid_pairs'][0][0]) != 'L2_basket']
```

Comment:
i'm not sure I would want to advertise `gid_to_type` ... let's think of a high level API